spec/ndx-extracellular-channels.extensions.yaml

```diff
@@ -0,0 +1,287 @@
+  default_name: probe_insertion
+  doc: Metadata about the insertion of a probe into the brain, which can be used to
+    determine the location of the probe in the brain.
+  attributes:
+  - name: reference
+    dtype: text
+    doc: Reference point for `insertion_position_in_mm` coordinates, e.g., "bregma
+      at the cortical surface".
+    required: false
+  - name: hemisphere
+    dtype: text
+    doc: The hemisphere ("left" or "right") of the targeted location of the optogenetic
+      stimulus site. Should be consistent with `insertion_position_in_mm.ml` coordinate
+      (left = ml < 0, right = ml > 0).
+    required: false
+  - name: depth_in_mm
+    dtype: float
+    doc: Depth that the probe was driven along `insertion_angle` starting from `insertion_position_in_mm`,
+      in millimeters.
+    required: false
+- neurodata_type_def: ChannelsTable
+  neurodata_type_inc: DynamicTable
+  default_name: ChannelsTable
+  doc: Metadata about the channels used in an extracellular recording from a single
+    probe.
+  attributes:
+  - name: reference_mode
+    dtype: text
+    doc: The reference mode used for the recording; e.g. 'external wire', 'common
+      reference'.
+    required: false
+  datasets:
+  - name: contact
+    neurodata_type_inc: DynamicTableRegion
+    doc: The row in a ContactsTable that represents the contact used as a channel.
+    quantity: '?'
+  - name: reference_contact
+    neurodata_type_inc: DynamicTableRegion
+    doc: The row in a ContactsTable that represents the contact used as a reference.
+    quantity: '?'
+  - name: filter
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: The filter used on the raw (wideband) voltage data from this contact, including
+      the filter name and frequency cutoffs, e.g., 'High-pass filter at 300 Hz.'
+    quantity: '?'
+  - name: estimated_position_in_mm
+    neurodata_type_inc: VectorData
+    dtype:
+    - name: ap
+      dtype: float
+      doc: Anteroposterior coordinate in mm, relative to `reference` (+ is anterior).
+    - name: ml
+      dtype: float
+      doc: Mediolateral coordinate in mm, relative to `reference` (+ is right).
+    - name: dv
+      dtype: float
+      doc: Dorsoventral coordinate in mm, relative to `reference` (+ is up).
+    doc: Stereotactic coordinates (AP, ML, DV) of the estimated contact position,
+      in millimeters. AP = anteroposterior coordinate in mm (+ is anterior). ML =
+      mediolateral coordinate in mm (+ is right). DV = dorsoventral coordinate in
+      mm (+ is up).Coordinates are relative to `reference`
+    quantity: '?'
+    attributes:
+    - name: reference
+      dtype: text
+      doc: Reference point for the position coordinates. e.g., "bregma at the cortical
+        surface".
+      required: false
+  - name: estimated_brain_area
+    neurodata_type_inc: VectorData
+    dtype: text
+    doc: The brain area of the estimated contact position, e.g., 'CA1'.
+    quantity: '?'
+  - name: actual_position_in_mm
```

Comment:
The word 'actual' here sounds a little odd. Maybe grants too much confidence? Even histology registration is still a form of estimation, just a higher resolution one

Perhaps 'verified'? or 'confirmed'?

As the opposite to 'estimated' (which I think is fine above), Google says 'confirmed' might be the best

Reply:
I changed "actual" to "confirmed" throughout.